### PR TITLE
faq: add entry about vsphere serial console

### PR DESCRIPTION
**Pull request by miabbott**

### CODE REVIEW

**Comment by miabbott** [comment]
@jcpowermac This is quick and dirty; feedback appreciated

File: docs/faq.md [hunk 1]
```diff
@@ -179,3 +179,9 @@ In the future the OpenShift installer will likely have support for this.
 ## Q: Can I use Driver Update Program disks with RHCOS?
 
 No, there is no supported mechanism for non-default kernel modules at this time, which includes [driver disks](https://access.redhat.com/articles/64322).
+
+## Q: How do I capture console logs from an RHCOS node running on vSphere?
+
+It's possible to write the serial console data directly to the VMFS volume.  You can do this by changing the Virtual Hardware settings of the VM to include a serial port that writes to a file (see [screenshot](https://raw.githubusercontent.com/openshift/os/docs/vsphere-settings.png)).
```
Comment (ashcrow) [inline docs/faq.md:185]: I'm not sure this link will work as expected. I _think_ a branch is required. As an example:

- No branch (404): https://raw.githubusercontent.com/openshift/os/docs/faq.md
- Branch: https://raw.githubusercontent.com/openshift/os/master/docs/faq.md

Comment (miabbott) [reply docs/faq.md:185]: Ah, yeah...quick and dirty like I said  :)

Will fix

**Comment by jcpowermac** [comment]
@miabbott maybe add a link to the vsphere docs?

https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-C6FBCF66-5796-4EE6-BF47-4DCAA9DCD1E3.html

I am unsure about my repo, haven't tested in quite a while, it _should_ work

**Comment by jcpowermac** [comment]
LGTM

**Review by ashcrow — APPROVED** [review]

**Pull request merged**